flathub.json

```diff
@@ -0,0 +1,4 @@
+{
+    "skip-icons-check": true,
```

Comment:
you should provide proper icons instead.

```suggestion
```

Reply:
Fixed